.github/workflows/prepare-for-minor-release.yml

```diff
@@ -0,0 +1,100 @@
+name: Prepare for minor release
+run-name: Update az-quickstart-scaffolding repository for new `${{ github.event.client_payload.release_branch_name }}` minor release branch.
+on:
+  repository_dispatch:
+    types: az_quickstart_prep_for_minor_release
+
+jobs:
+  prepare_for_new_minor_release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up git
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+
+      - name: Setup PHP with composer v2
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.1'
+          tools: composer:v2
+        
+      - name: Checkout az-quickstart-scaffolding
+        uses: actions/checkout@v4
+
+      - name: Create new release branch and pushing it back to the repo.
+        run: |
+          git checkout -b ${{ github.event.client_payload.release_branch_name }}
+          # git push -u origin ${{ github.event.client_payload.release_branch_name }}
+
+      - name: Create new branch so we can make a pull request against `${{ github.event.client_payload.release_branch_name }}`
+        run: |
+          git checkout -b ${{ github.event.client_payload.release_branch_name }}-prepare-for-minor-release
+
+      - name: Update the az-digital/az_quickstart composer.json requirement
+        run: | 
+          composer require az-digital/az_quickstart ${{ github.event.client_payload.this_release_branch_alias_constraint }} --no-update --no-scripts --no-interaction
+
+      - name: Add composer.json change to commit for main branch PR
+        run: |
+          git diff
+          git add composer.json
+          git diff
+          git commit -m "Update Quickstart version constraint to ${{ github.event.client_payload.this_release_branch_alias_constraint }}"
```

Comment:
Since we are already referencing the next minor release in main, we don't need this anymore.
```suggestion
```